src/screens/NewAnalytics/Graphs/LineGraph/LineGraphTypes.res

```diff
@@ -2,6 +2,12 @@ type \"type" = string
 type spacingLeft = int
 type spacingRight = int
 
+type point = {color: string, x: string, y: float}
+type pointFormatter = {points: array<point>}
+
+external asTooltipPointFormatter: Js_OO.Callback.arity1<'a> => pointFormatter => string =
```

Comment:
Add this in Identity file and `Js_OO.Callback.arity1` what it is is this some required can't we make it generic?? like `a